### DISCUSSION TITLE
quick-tap-ms not working for BSPC after editing the

### DISCUSSION THREAD

**Discussion by suriaribeiro** [manna-harbour/miryoku#216] [Q&A] [answered]
After editing ...[miryoku/custom_config.h](https://github.com/suriaribeiro/miryoku_zmk/blob/aa72c73d7e3cf5c8abbf764f8a5081a50ddb7fef/miryoku/custom_config.h) BSPC not longer works as a BSPC when double tap and held.

What do I need to update so that I can enable this functionality after customising the base layer?   

thanks in advance for any support you can provide.

**Comment by manna-harbour** [comment, marked as answer]
`quick-tap-ms` is not enabled by default so that you can hold a mod or layer change immediately after tapping the same key. To enable it you could add it to https://github.com/manna-harbour/miryoku_zmk/blob/master/miryoku/miryoku_behaviors.dtsi.

To auto repeat Backspace you can use the Backspace on Nav, and / or combine with Control to delete whole words.
